css-positioning/absolute.css

```diff
@@ -0,0 +1,39 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 50px;
```

Comment:
You want the emoji in the top right of the meme, right? Moving it from the left to the right edge is a big movement. You should try moving it from the right side, it will allow you to have a much smaller unit 👍